### PR TITLE
fix wrongly inverted icon

### PR DESCRIPTION
**Pull request by szaimen**
Fix https://github.com/nextcloud/activity/issues/1462

| Before | After |
|---|---|
| image | image |

<details>
<summary>For my own testing</summary>

```
docker run -it --rm ^
--name nextcloud-easy-test ^
-p 8443:443 ^
-e ACTIVITY_BRANCH=enh/noid/fix-icon ^
-e SERVER_BRANCH=master ^
--volume="nextcloud_easy_test_npm_cache_volume:/var/www/.npm" ^
ghcr.io/szaimen/nextcloud-easy-test:latest

```

</details>

### CODE REVIEW

**Comment by szaimen** [comment]
Doesn't seem to work yet...

File: src/views/ActivityAppNavigation.vue [hunk 1]
```diff
@@ -127,7 +127,6 @@ async function copyRSSLink() {
 	}
 
 	.navigation-icon {
-		filter: var(--background-invert-if-dark);
```
Comment (szaimen) [inline src/views/ActivityAppNavigation.vue:130]: might simply need to make an exception for if selected...

**Comment by szaimen** [comment]
Works now and ready for review :)

**Review by artonge — APPROVED** [review]

**Review by jancborchardt — APPROVED** [review]
Looks very nice! :)

**Pull request merged**